lib/ethpy/ethpy/eth_config.py

```diff
@@ -41,15 +48,15 @@ def build_eth_config() -> EthConfig:
     # Look for and load local config if it exists
     load_dotenv("eth.env")
 
-    artifacts_url = os.getenv("ARTIFACTS_URL")
-    rpc_url = os.getenv("RPC_URL")
+    artifacts_uri = os.getenv("ARTIFACTS_URI")
+    rpc_uri = os.getenv("RPC_URI")
     abi_dir = os.getenv("ABI_DIR")
 
     arg_dict = {}
-    if artifacts_url is not None:
-        arg_dict["ARTIFACTS_URL"] = artifacts_url
-    if rpc_url is not None:
-        arg_dict["RPC_URL"] = rpc_url
+    if artifacts_uri is not None:
+        arg_dict["ARTIFACTS_URI"] = artifacts_uri
```

Comment:
Bug is still here, `EthConfig` defines the member variables as lowercase right?

Reply:
Oops, yes good catch. Also a good example of why we need to bump up our test coverage -- class initializations should be tested pretty thoroughly.

Reply:
I think this is passing CI because we always explicitly set an eth config in tests. I assume this will break when actually running the pipeline.